test/hashmaps/test_maps.fypp

```diff
@@ -24,7 +24,7 @@ module test_stdlib_chaining_maps
     integer, parameter        :: test_size = rand_size*4
     integer, parameter        :: test_16 = 2**4
     integer, parameter        :: test_256 = 2**8
-
+    integer, parameter        :: key_types = 2
```

Comment:
Here could you add a comment along the lines of `! The key types are int8 and int32`. I think that's the idea?

Reply:
Tried to add in comments to explain this.